### PR TITLE
Emoji support for chats

### PR DESCRIPTION
**Pull request by m1x0n**
This PR adds naive emoji support for chats by introducing emoji picker button.
Emoji picker is provided by https://github.com/ealush/emoji-picker-react .

Partially solves https://github.com/VladimirKhil/SIOnline/issues/94 .

### CODE REVIEW

File: src/components/common/ChatInputEmojiPicker.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { useState } from 'react';
+import EmojiPicker, { EmojiClickData, EmojiStyle, SkinTones } from 'emoji-picker-react';
+import './ChatInputEmojiPicker.css';
+
+interface ChatInputEmojiPickerProps {
+	onEmojiClick: (emojiData: EmojiClickData, e: MouseEvent) => void;
+}
+export default function ChatInputEmojiPicker(props: ChatInputEmojiPickerProps): JSX.Element {
+	const BUTTON_HINT = 'Pick an emoji';
```
Comment (VladimirKhil) [inline src/components/common/ChatInputEmojiPicker.tsx:10]: Please move this constant to localization object.

**Review by VladimirKhil — CHANGES_REQUESTED** [review]
Only one string needed to be localized. All other changes LGTM.

File: src/components/common/ChatInputEmojiPicker.tsx (hunk 1 above)
Comment (m1x0n) [reply src/components/common/ChatInputEmojiPicker.tsx]: Done

**Review by VladimirKhil — COMMENTED** [review]
Great, thank you a lot!

**Review by VladimirKhil — APPROVED** [review]

**Pull request merged**